TripKit/Classes/server/TKBuzzInfoProvider.swift

```diff
@@ -99,19 +100,30 @@ extension TKBuzzInfoProvider {
    Asynchronously fetches transit alerts for the provided region using Rx.
    */
   public class func rx_fetchTransitAlerts(forRegion region: SVKRegion) -> Observable<[API.Alert]> {
+    return rx_fetchTransitAlertMappings(forRegion: region)
+      .map { $0.map {$0.alert} }
+      .catchErrorJustReturn([])
+  }
+  
+  public class func rx_fetchTransitAlertMappings(forRegion region: SVKRegion) -> Observable<[API.AlertMapping]> {
     let paras: [String: Any] = [
-      "region": region.name as Any
+      "region": region.name
     ]
     
     return SVKServer.shared.rx
       .hit(.GET, path: "alerts/transit.json", parameters: paras, region: region)
-      .map { (_, _, data) -> [API.Alert] in
+      .map { (_, _, data) -> [API.AlertMapping] in
         let decoder = JSONDecoder()
-        guard let data = data, let response = try? decoder.decode(AlertsTransitResponse.self, from: data) else { return [] }
-        return response.alerts.map { $0.alert }
-    }
+        decoder.dateDecodingStrategy = .secondsSince1970
```

Comment:
Please add a TODO here that this will need fixing for JSON `v >= 13` as we'll then use ISO8601.